ui/client/containers/EnsureArray.tsx

```diff
@@ -0,0 +1,5 @@
+import {isArray} from "lodash"
```

Comment:
I wonder.. sometimes we use camelcase start with big letter anothertime we use it by start small letter? Can you tell my why? Can we assume that when we export only methods then we start with small letter?  Alsow I think this util shoul be in package like utils.

It's only array tools -> it should named `arrayUtils`.

Reply:
changed to arrayUtils